### PR TITLE
Commit Codes

### PR DESCRIPTION
**Pull request by BrandonKlotz**

### CODE REVIEW

File: brand.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en-us">
   <head>
     <meta charset="UTF-8">
-    <title>Citizen Labs</title>
+    <title>Citizen Labs by citizenlabsgr</title>
```
Comment (jacebrowning) [inline brand.html]: I think you want to start a new branch off `master`. In this commit you're reverting a change that was already made.

Comment (BrandonKlotz) [inline brand.html]: Should I close this Pull Request and then start a new branch from the current master and add the changes?

Comment (BrandonKlotz) [inline brand.html]: Or since it is one line just change it to the updated version and commit so there is no difference?

Comment (jacebrowning) [inline brand.html]: That works.

**Pull request merged**